packages/tools/devtools/devtools-view/src/components/SettingsView.tsx

```diff
@@ -107,7 +117,7 @@ export function SettingsView(): React.ReactElement {
 			<div className={styles.option}>
 				<label className={styles.label}>Select theme</label>
 				<Dropdown
-					placeholder="Theme"
+					placeholder={themeReadableOption[themeInfo.name as ThemeName]}
```

Comment:
Hmm are we sure this is the correct way? Does `Dropdown` have a prop for "currently selected" or something along those lines?

Reply:
I initially tried using `useState` to store the value that has been selected by the `<Dropdown>` option. So something like:
- Create `const [selectedTheme, setSelectedTheme] = useState("Theme")`
- Trigger `onClick()` event to set the `selectedTheme` correspondingly
- Render `selectedTheme` in our `placeholder`

However, even with this change, the problem persisted and I believe this is due to the way how `View` component (from the `DevtoolsView`) selects its "to-be" rendered component through switch case and initializes the state back to the default selected placeholder. 

Therefore, I instead chose to use `React.context()` which is propagated gloabally from parent to child component to access the currently selected theme.

Reply:
But that's separate from my point. Using the `placeholder` property of the `Dropdown` is what I find strange. There must be a way to bind the current value to the actual selected value in the dropdown, not just "fake it" by writing it to the placeholder text which is only supposed to appear when there's no selection. In our app I think "no selection" doesn't make sense. There's always a selected theme; we decided that Dark is the default, and that should mean that when the user comes to the Settings view, that theme should already be selected.

Reply:
Change using to `value` as it seems more logical than using `placeholder`. But I am still skeptical (and I think this goes along with your point) of using `themeReadableOption[themeInfo.name as ThemeName]` as it feels like our `SettingsView` now has two different sources of data.

Reply:
I agree. The problem there is that when using `setTheme()` in the switch statement we're setting values different than the ones from the `ThemeOption` enum, so then we have to "translate back" from those to the ones in `ThemeOption`, which you did with `themeReadableOption`. If we update the `setTheme()` lines to something like 

```typescript
setTheme({
	name: ThemeOption.Light, // instead of the string "light"
	theme: webLightTheme,
});
```

then I think the code will be more straightforward. I'm only slightly concerned about the fact that the value of one of the options in `ThemeOption` is "High Contrast" with a space in the middle but I think it should be fine... If you do this change let's just make sure that theme selection still works for all three themes.